models/depth_camera/depth_camera.sdf.jinja

```diff
@@ -24,6 +25,38 @@
         </geometry>
       </visual>
       <sensor name="depth_camera" type="depth">
+        {% if ros2_distro in ['dashing', 'eloquent', 'foxy'] %}
+        <always_on>0</always_on>
+        <update_rate>20</update_rate>
+        <camera name="camera_name">
+          <horizontal_fov>1.02974</horizontal_fov>
+          <image>
+            <format>R8G8B8</format>
+            <width>64</width>
+            <height>48</height>
+          </image>
+          <clip>
+            <near>0.5</near>
+            <far>18</far>
+          </clip>
+          <distortion>
+            <k1>0.0</k1>
+            <k2>0.0</k2>
+            <k3>0.0</k3>
+            <p1>0.0</p1>
+            <p2>0.0</p2>
+            <center>0.5 0.5</center>
+          </distortion>
+        </camera>
+
+        <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
```

Comment:
Is this plugin specific to ROS2?

Reply:
Yes, for using depth camera in ROS2, "libgazebo_ros_camera.so" should be used, instead of "libgazebo_ros_depth_camera.so" or "libgazebo_ros_openni_kinect.so". (According to [Additional steps for gazebo_ros_depth_camera](https://github.com/ros-simulation/gazebo_ros_pkgs/wiki/ROS-2-Migration:-Camera#gazebo_ros_depth_camera)).

The name of parameters are also different in ROS1 and ROS2.